.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ commands:
           at: /tmp
       - run:
           name: 'Maybe Gracefully Halt'
-          command: if [[ -f "/tmp/workspace/.CI_GRACEFULLY_HALT" ]]; then echo "Gracefully halting this job."; circleci-agent step halt; fi
+          command: if ls .CI_GRACEFULLY_HALT_* 1>/dev/null 2>&1; then echo "Gracefully halting this job."; circleci-agent step halt; fi
```

Comment:
Amazing how complex it gets to do this simple thing...

Reply:
Yeah, complex enough that I forgot about the `/tmp/workspace/` prefix. Fixing this in https://github.com/ampproject/amphtml/pull/33691.